compiler/optimizer/VPHandlers.cpp

```diff
@@ -1700,10 +1700,16 @@ static const char *getFieldSignature(OMR::ValuePropagation *vp, TR::Node *node,
 static bool addKnownObjectConstraints(
    OMR::ValuePropagation *vp, TR::Node *node, bool isGlobal)
    {
+   // 'addKnownObjectConstraints` has a large overhead for remote compilations
```

Comment:
While this is a valid short term solution, I assume there will be investigation done on an ongoing basis to re-enable this logic under JIT server ?

Reply:
I will look at it, though any code that looks inside objects cannot be processed at the server and needs a message to the client. I am unsure about the value of caching, since the known object table is only valid for the current compilation.

Reply:
The reason I mention is that the known object table in general can be useful in other contexts, e.g. method handle performance or other cases of code passing in a known object (a string or class constant) to APIs that can be optimized.